.github/workflows/labeler.yml

```
@@ -0,0 +1,17 @@
+name: "Pull Request Labeler"
+on:
+  pull_request_target:
+    types: [opened, synchronized, reopened]
```

actionlint (1.7.12)

[error] invalid activity type "synchronized" for "pull_request_target" Webhook event. available types are "assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "demilestoned", "dequeued", "edited", "enqueued", "labeled", "locked", "milestoned", "opened", "ready_for_review", "reopened", "review_request_removed", "review_requested", "synchronize", "unassigned", "unlabeled", "unlocked"

(events)